src/routes/spellbook/+page.svelte

```diff
@@ -78,9 +101,52 @@
             addSpell(spells[0]);
             return;
         }
+        await searchSpells();
+    }
 
+    async function handleLevelUpdate(event: CustomEvent) {
+        selectedLevel = event.detail.level;
         await searchSpells();
     }
+
+    async function handleClassUpdate(event: CustomEvent) {
+        selectedClass = event.detail.class;
+        await searchSpells();
+    }
+
+
+    function handleLevelSort() {
+        if(isSortingByAscending){
+            spells = spells.sort((a, b) => b.level_int - a.level_int);
+            isSortingByAscending=false;
+        }
+        else{
+            spells = spells.sort((a, b) => a.level_int - b.level_int);
+            isSortingByAscending=true;
+        }
```

Comment:
Du kan undgå at bruge else ved at bruge en early return. 

```suggestion
        if (isSortingByAscending) {
            spells = spells.sort((a, b) => b.level_int - a.level_int);
            isSortingByAscending = false;
            return;
        }
        
        spells = spells.sort((a, b) => a.level_int - b.level_int);
        isSortingByAscending = true;
```